### PR TITLE
Typo in Environments.php

### PR DESCRIPTION
**Pull request by ghost**
Adding the 's' on environments so that is doesn't result in 404

### CODE REVIEW

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]
Please fix the tests. :)

**Comment by ghost** [comment]
Umm,

Dude! I fixed ^your^ typo. I didn't do anything to the tests. If the tests are now broken that means your tests were BS to begin with.

I am using this library to make something for my work and I don't have time for amatuer hour shenanigans.

I would have hoped this library ^WAS TESTED^ and it didn't have a typo in it. Alas, here we are...

I am more than happy to make a composer patch and fix your problem on the fly, than deal with your laziness. Seriously, I have already done 100x more effort than I should have done and tried to help you. Your response to that, "Oh could you please do more?" 🤦 

Good Day Sir!
-- R

**Comment by ahinkle** [comment]
> I am more than happy to make a composer patch and fix your problem on the fly, than deal with your laziness. Seriously, I have already done 100x more effort than I should have done and tried to help you. Your response to that, "Oh could you please do more?" 🤦

@brownrl, 

No one owes you anything.  These maintainers (like @GrahamCampbell) maintain several, if not hundreds of open-source packages. You could have taken less time (in fact a matter of mere seconds) to change a single character on the test instead of posting these unhelpful and rude comments.

**Comment by mateusjatenee** [comment]
> Umm,
> 
> 
> 
> Dude! I fixed ^your^ typo. I didn't do anything to the tests. If the tests are now broken that means your tests were BS to begin with.
> 
> 
> 
> I am using this library to make something for my work and I don't have time for amatuer hour shenanigans.
> 
> 
> 
> I would have hoped this library ^WAS TESTED^ and it didn't have a typo in it. Alas, here we are...
> 
> 
> 
> I am more than happy to make a composer patch and fix your problem on the fly, than deal with your laziness. Seriously, I have already done 100x more effort than I should have done and tried to help you. Your response to that, "Oh could you please do more?" 🤦 
> 
> 
> 
> Good Day Sir!
> 
> -- R
> 
> 
> 
> 

Fuck off, if anything you're the lazy one by "using the package". Don't do any effort then and implement the solution from scratch.

**Comment by ghost** [comment]
Hey @ahinkle 

I did change the test. Jerky McJerkFace!

For the record I didn't ask for anything. I came here and made a fork to try to help someone. Instead of saying "Hey, Thanks for the help. I appreciate it." I get "oooh THE TESTS!". 

The tests don't actually test if the thing that they are supposed to do works. Instead the test replicates the typo and only tests if the final url path matches .... wait for it.... the typo!

Forgive me for trying to help people.

**Comment by kife-design** [comment]
@brownrl so you use his package in one of your projects, saving you hours of development time compared to programming it yourself.  You fix a typo and flame the author for asking you to fix the tests? Talk about having no shame at all.

**Comment by ghost** [comment]
Rest assured, I'm not using it anymore.

**Comment by drupol** [comment]
So sad to spoil the company you're working for like that. I know @mathiasselleslach and I doubt that he would agree with such comments towards the Open Source community. Shame on you @brownrl.

File: src/Api/Environments.php [hunk 1]
```diff
@@ -54,7 +54,7 @@ public function create($project_id, array $parameters = [])
         $resolver->setDefined('external_url')
```
Comment (WyriHaximus) [inline src/Api/Environments.php:54]: There is a typo in your commit message, `Environemnts` should be `Environments`

Comment (matthewtrask) [reply src/Api/Environments.php:54]: there is also a typo in his reply to Graham. It's `amateur` not `amatuer`

**Pull request closed without merging**